zulipterminal/ui.py

```diff
@@ -77,6 +77,24 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Any],
+                             state: Optional[int],
+                             is_truncated: bool) -> None:
+        # Wrap by space.
+        suggestions = list(map(lambda x: ' ' + x + ' ', suggestions))
```

Comment:
I see you moved a lot of the suggestions code into the conditional block (if/else) - it looks easier to read :+1:

However, we only need this line if there are suggestions, so can it move into the block too?